libs/core/src/components/pds-tooltip/docs/pds-tooltip.mdx

```diff
@@ -0,0 +1,116 @@
+import { DocArgsTable, DocCanvas } from '@pine-ds/doc-components';
+
+# Tooltip
+
+Tooltips display helpful popup text when users hover over or keyboard focus on an element.
+
+## Guidelines
+### Usage
+#### When to use
+- To describe icon-only buttons
+- When adding more context or explanation to an element is helpful, but not essential
+
+#### When not to use
+- Do not use a tooltip when information is essential to task completion.
+That information should remain visible.
+
+## Properties
+
+<DocArgsTable componentName="pds-tooltip" />
+
+## Variants
+### Trigger
+Whether your trigger is text or an HTML element, it will be placed in the default slot between the opening and closing tags.
+
+<DocCanvas client:only
+  mdxSource={{
+    react: `
+      <PdsTooltip content="this is tooltip content" has-arrow="true" placement="bottom-start">
+        text
+      </PdsTooltip>
+      <PdsTooltip content="this is tooltip content" has-arrow="true" placement="right-start">
+        <PdsButton variant="secondary">Button</pds-button>
+      </PdsTooltip>
+    `,
+    webComponent: `
+      <pds-tooltip content="this is tooltip content" has-arrow="true" placement="bottom-start">
+        text
+      </pds-tooltip>
+      <pds-tooltip content="this is tooltip content" has-arrow="true" placement="right-start">
+        <pds-button variant="secondary">Button</pds-button>
+      </pds-tooltip>
+    `
+}}>
+  <pds-tooltip content="this is tooltip content" has-arrow="true" placement="bottom-start">
+    text
+  </pds-tooltip>
+  <pds-tooltip content="this is tooltip content" has-arrow="true" placement="right-start">
+    <pds-button variant="secondary">Button</pds-button>
+  </pds-tooltip>
+</DocCanvas>
+
+### Content
+#### Text Content
+Text-based content is generated using the `content` prop. Use this option only when the
+tooltip content is expected to be a string.
+
+<DocCanvas client:only
+  mdxSource={{
+    react: '<PdsTooltip content="this is tooltip content" hasArrow="true" placement="right">text</PdsTooltip>',
+    webComponent: '<pds-tooltip content="this is tooltip content" has-arrow="true" placement="right">text</pds-tooltip>'
+  }}>
+  <pds-tooltip content="this is tooltip content" has-arrow="true" placement="right">text</pds-tooltip>
+</DocCanvas>
+
+#### HTML Element Content
+Tooltip content may contain HTML elements. In these cases,
+enable the `htmlContent` property and provide your element in the `content` slot.
+
+<DocCanvas client:only
+  mdxSource={{
+    react: `
+      <PdsTooltip hasArrow="true" placement="right">
+        text
+        <div slot="content">
+          <p>this is a sentence in a tooltip.</p>
+          <p>this is a sentence in a tooltip.</p>
+        </div>
```

Comment:
Can prob be handled in a follow up but looks like the HTML content is inheriting some colors from somewhere, which is odd...

![Screenshot 2023-12-06 at 4 42 48 PM](https://github.com/Kajabi/pine/assets/1175111/d9eea555-e972-4224-a1d1-e7725692b98e)

Reply:
Here is the style that is affecting it. I'm just not sure as to why. Looking into this now
![image](https://github.com/Kajabi/pine/assets/1633290/af85df0c-719c-4b6d-8427-898b9cff7566)

Reply:
seems odd that what I am assuming are the storybook styles would affect/override the component styles.